packages/command/messages/flags.json

```diff
@@ -23,6 +23,7 @@
   "MissingOrInvalidFlagDescription": "The flag %s is missing the description attribute, or the description is not a string.",
   "InvalidFlagChar": "The flag %s's char attribute must be one alphabetical character long.",
   "InvalidFlagName": "The flag %s's name must be a lowercase string that may contain numbers and hyphens.",
+  "FormattingMessageArrayValue": "Must only contain valid values.",
```

Comment:
It's sort of tricky to do it here, since the validator can accept either regex (which can be ugly) or a function(which is opaque from a stringification point of view).  I imagine the best practice would be for the command author to describe the range of valid values in the description?